proto/serverless/instrumentation/tags/v1/tags.proto

```diff
@@ -0,0 +1,306 @@
+  optional AwsSdkSnsTags aws_sdk_sns = 105;
+
+
+  // The HTTP Tags.
+  optional HttpTags http = 106;
+
+  // The HTTPS Tags.
+  optional HttpTags https = 107;
+
+  // The Express Tags.
+  optional ExpressTags express = 108;
+
+  // The AWS Trace Context Tags.
+  optional AwsTags aws = 109;
+}
+
+message SlsSdkTags {
+  // A Serverless Platform OrgId.
+  string org_id = 1;
+  // The platform that was instrumented. Currently Lambda is the only supported platform.
+  string platform = 2;
+  // The service that was instrumented. For Lambda this will be the function name by default.
+  string service = 3;
+  // The region that instrumentation was performed in. This is used to determine which Serverless Ingest API to use.
+  string region = 4;
+
+  message SdkTags {
+    // The Name of the Serverless SDK used to instrument.
+    string name = 1;
+    // The version of the Serverless SDK used to instrument.
+    string version = 2;
+  }
+
+  SdkTags sdk = 5;
+
+  // An optional environment that can be attached.
+  optional string environment = 16;
+  // An optional namespace that can be attached.
+  optional string namespace = 17;
+
+
+}
+
+message AwsTags {
+  // The AWS Account ID that instrumentation was performed in.
+  string account_id = 1;
+  // The ARN of the instrumented AWS resource.
+  string resource_arn = 2;
+
+  // The optional X-Ray TraceId from AWS. This will only be available if X-Ray was enabled in the instrumented Account & Service.
+  optional string xray_trace_id = 16;
+}
+
+message AwsApiGatewayTags {
+  // The AWS Account ID of the API GW.
+  string account_id = 1;
+  // The unique id used for the API GW.
+  string api_id = 2;
+  // The stage of the API GW endpoint that was called.
+  string api_stage = 3;
+
+  message AwsApiGatewayRequestTags {
+    // The unique API GW Request ID.
+    string id = 1;
+    // The request time in milliseconds from epoch.
+    fixed64 time_epoch = 2;
+    // The HTTP protocol of the request.
+    string protocol = 3;
+    // The domain name of the request.
+    string domain = 4;
+    // JSON string containing Request Headers
+    string headers = 5;
+    // The HTTP method of the request.
+    string method = 6;
+    // The HTTP Path of the request.
+    string path = 7;
+
+
+    // JSON string containing Request Path Parameters
+    optional string path_parameters = 16;
+    // JSON string contain Query String Parameters
+    optional string string_parameters = 17;
+  }
+
+  AwsApiGatewayRequestTags request = 15;
+
+}
+
+message AwsLambdaTags {
+  // The architecture of the Lambda function, currently either amd64 or arm64.
+  string arch = 1;
+  // Was the invocation a cold start?
+  bool coldstart = 2;
+  // The event type for the invocation.
+  string event_type = 3;
+  // The event source for the invocation.
+  string event_source = 4;
+  // The Log Group for the Lambda Function.
+  string log_group = 5;
+  // The Log Stream for the invocation.
+  string log_stream_name = 6;
```

Comment:
How? Just because they are in cloudwatch does not mean we can easily source them everywhere. In general if this data is available in the function context, then we should use it there.

Reply:
> How?

When processing logs it's clear which log stream it comes from

>  In general if this data is available in the function context, then we should use it there.

Not really, see my comment at https://github.com/serverless/console/pull/139#discussion_r947871485 (I've also raised that in Jira for wider discussion)